admin/cluster_setup.py

```diff
@@ -159,6 +174,15 @@ def main(reactor, args, base_path, top_level):
     try:
         yield runner.ensure_keys(reactor)
         cluster = yield runner.start_cluster(reactor)
+
+        managed_config_file = options['cert-directory'].child("managed.yaml")
+        managed_config = dict()
+        managed_config.update(options['config'])
+        managed_config.update(generate_managed_config(cluster))
+        managed_config_file.setContent(
+            yaml.safe_dump(managed_config, default_flow_style=False)
+        )
```

Comment:
:question: Maybe make this a separate function that takes `managed_config_file`, `options['config']`, and `cluster` as parameters?